Library/Homebrew/compat/download_strategy.rb

```diff
@@ -0,0 +1,16 @@
+class AbstractFileDownloadStrategy
+  # TODO: This can be removed after a month because downloads
```

Comment:
I don't agree with this; I'd rather we kept around something indefinitely as there's plenty of things that get updated less than once a month. Can the same logic not live in `brew update` indefinitely?